src/api/tasks.js

```diff
@@ -18,6 +18,16 @@ export const getTasksReq = (pursuanceId) => {
     });
 }
 
+export const deleteTaskReq = (taskGid) => {
+  return postgrest.deleteJSON(`/tasks?gid=eq.${taskGid}`,)
+    .then((ok) => {
+      return ok;
+    })
+    .catch((err) => {
+      console.log("Error deleting task '${taskGid}':", err);
```

Comment:
@ajvb ES6 string interpolation only works if you use backquotes rather than single or double quotes to define the string, so do 

> `"Error deleting task '${taskGid}':"`

Reply:
It's coming :) was just pushing up a merge.